src/libraries/System.Net.Http/src/System/Net/Http/Headers/TransferCodingHeaderValue.cs

```diff
@@ -154,7 +154,7 @@ public override bool Equals(object obj)
         public override int GetHashCode()
         {
             // The value string is case-insensitive.
-            return StringComparer.OrdinalIgnoreCase.GetHashCode(_value) ^ NameValueHeaderValue.GetHashCode(_parameters);
+            return StringComparer.OrdinalIgnoreCase.GetHashCode(_value!) ^ NameValueHeaderValue.GetHashCode(_parameters);
```

Comment:
Suppressed not because of value is not null, but i suspect this method accepts null a i have seen similar situation several times before, i have tried to check implementation but lost within native code ...

Reply:
Do you mean that GetHashCode nullable annotations is wrong?

Reply:
By having another look I have mistaken GetHashCode method of another type, and looks `value` might even should be non null as non empty contractors not allowing null, but I cannot follow empty constructor usage completely, so I think I will leave it as nullable but assert for null here

Reply:
Well actually internal empty constructor only used for internal parser and `_value` set non null after usage, so this `_value` should be annotated non null